src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,30 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todos';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => {
+        const {
+          id,
+          completed,
+        } = todo;
```

Comment:
You can write it in one line in this case
```suggestion
        const { id, completed } = todo;
```